README.md

```diff
@@ -6,11 +6,14 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
 
-Datafuse is a free Cloud-Native Analytics DBMS(Inspired by ClickHouse) implemented in Rust.
+Datafuse is a Cloud-Native Distributed Data Processing & Analysis DBMS implemented in Rust.
 
 Inspired by [ClickHouse](https://github.com/ClickHouse/ClickHouse) and powered by [Arrow](https://github.com/apache/arrow).
 
-## Features
+## Principles
+
+* **Fearless**
+  - No data races, No unsafe, Minimize unhandled errors
```

Comment:
No unsafe? I see lots of code in arrow using unsafe

Reply:
Hmm, I assume that 'fearless unsafe' is not Rust unsafe.